### ISSUE TITLE
CPU temp is extraordinarily high

### ISSUE THREAD

**Issue by Spyromaniac31** [hoobs-org/HOOBS#400]
**Description**
The CPU temperature is in the 100s (F), which is obviously unhealthy for the Pi
**Version**
3.1.17

**Did you upgrade**
Yes, but the issue occurred before upgrade

**Previous version**
3.1.15
**Did you originally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Not sure what I upgraded to HOOBS 3 from, but I didn't always have the issue with HOOBS 3

**What device are you using?**
HOOBS bootable image on a Raspberry Pi Zero W

**List your plugins**
Tplink Smarthome
myQ2
Smartglass
Tuya Web

**Post your config**
This can be found in the interface, Configuration -> Advanced or you can get it via SSH `cat ~/.hoobs/etc/config.json`

* Remove any passwords from the config file. This is public.

```
{
    "server": {
        "port": 8080,
        "autostart": 0,
        "home_setup_id": "X-HM://0023ISYWYE0L5",
        "polling_seconds": 5,
        "origin": "*"
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "hoobs-dark",
        "locale": "en",
        "temp_units": "fahrenheit",
        "country_code": "US",
        "postal_code": "87106"
    },
    "bridge": {
        "name": "HOOBS",
        "username": "CC:22:3D:E3:CE:30",
        "port": 51826,
        "pin": "031-45-154"
    },
    "description": "This file is used for plugin configuration.",
    "ports": {},
    "accessories": [
        {
            "accessory": "Smartglass",
            "name": "Xbox One",
            "liveid": "********************",
            "consoleip": "192.168.0.36"
        }
    ],
    "platforms": [
        {
            "platform": "MyQ2",
            "email": "jimmy@jpontzer.net",
            "password": "*************"
        },
        {
            "platform": "TplinkSmarthome",
            "name": "TplinkSmarthome"
        },
        {
            "platform": "TuyaWebPlatform",
            "name": "TuyaWebPlatform",
            "options": {
                "username": "henry@jpontzer.net",
                "password": "********",
                "countryCode": "1",
                "platform": "tuya"
            }
        }
    ]
}
```

**Post your log**
You can get the log from the intercafe. This is in the Log section.

```
[1/3/2020, 10:01:21 AM] HOOBS listning on port 8080.
[Homebridge] [1/3/2020, 10:01:29 AM] Loaded config.json with 1 accessories and 3 platforms.
[Homebridge] [1/3/2020, 10:01:30 AM] Loaded plugin: "homebridge-myq2"
[Homebridge] [1/3/2020, 10:01:30 AM] Registering platform "homebridge-myq2.MyQ2"
[Homebridge] [1/3/2020, 10:01:31 AM] Loaded plugin: "homebridge-smartglass"
[Homebridge] [1/3/2020, 10:01:31 AM] Registering accessory "homebridge-smartglass.Smartglass"
[Homebridge] [1/3/2020, 10:01:32 AM] Loaded plugin: "homebridge-tplink-smarthome"
[Homebridge] [1/3/2020, 10:01:32 AM] Registering platform "homebridge-tplink-smarthome.TplinkSmarthome"
[Homebridge] [1/3/2020, 10:01:34 AM] Loaded plugin: "homebridge-tuya-web"
[Homebridge] [1/3/2020, 10:01:34 AM] Registering platform "homebridge-tuya-web.TuyaWebPlatform"
[Homebridge] [1/3/2020, 10:01:34 AM] Loading 3 platforms...
[Homebridge] [1/3/2020, 10:01:34 AM] [MyQ2] Initializing MyQ2 platform...
[Homebridge] [1/3/2020, 10:01:34 AM] [TplinkSmarthome] Initializing TplinkSmarthome platform...
[Homebridge] [1/3/2020, 10:01:35 AM] [TplinkSmarthome] homebridge-tplink-smarthome v4.0.1, node v10.16.0, homebridge v0.4.50
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Initializing TuyaWebPlatform platform...
[Homebridge] [1/3/2020, 10:01:35 AM] Loading 1 accessories...
[Homebridge] [1/3/2020, 10:01:35 AM] [Xbox One] Initializing Smartglass accessory...
[Homebridge] [1/3/2020, 10:01:35 AM] [Xbox One] Registering Television Service...
[Homebridge] [1/3/2020, 10:01:35 AM] [Xbox One] Registering Information Service...
[Homebridge] [1/3/2020, 10:01:35 AM] [Xbox One] Registering Volume Service...
[Homebridge] [1/3/2020, 10:01:35 AM] [Xbox One] Registering Key Service...
[Homebridge] [1/3/2020, 10:01:35 AM] [Xbox One] Registering InputSource Service...
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Henry 1] 04200320dc4f222e943c_1 2d0c8c38-9f5b-410a-b26f-5a14a833d445
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Henry 2] 04200320dc4f222e943c_2 d7d6a5ab-86ee-40e5-a67a-19e62fd89ff0
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Henry 3] 04200320dc4f222e943c_3 0e9e2b27-8d89-4c4a-bf02-2a3a56df1179
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Henry 4] 04200320dc4f222e943c_4 a5edeb6b-b0e0-4923-ad20-4b66f94b84e7
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Henry USB] 04200320dc4f222e943c_5 05936f33-4676-4519-a53d-1d9fef5ca9dc
[Homebridge] [1/3/2020, 10:01:35 AM] [TplinkSmarthome] Configuring cached accessory: [Smart Outlet] 80062BDC6455BAAC551F9C6E9458DE991BA17CD8 e45fcd37-fbe7-4051-bb23-eaf47260e1c1
[Homebridge] [1/3/2020, 10:01:35 AM] [TplinkSmarthome] Configuring cached accessory: [Kitchen Under-Cabinet Lights] 8006F2667FD8CB480360E856E8B0C0E11B6D1481 bb0cdc7e-47fb-4541-a2d8-7fcb52618833
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Family 1] 077550453c71bf3e720c_1 e4ce4d75-fbd4-4914-bcb0-3fb3b17cf474
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Family 2] 077550453c71bf3e720c_2 c1a505da-7679-4e43-aac0-5de467bc6849
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Family 3] 077550453c71bf3e720c_3 af6bad5e-6213-4dde-8e94-0704e7e7a132
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Family 4] 077550453c71bf3e720c_4 cd6a38e7-4e1a-45b1-9b1e-5645456efce2
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Family USB] 077550453c71bf3e720c_7 8b7507af-af87-4f20-9798-64a8f0197a27
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Study 1] 077550453c71bf3e748d_1 876a3c05-666a-4103-87cb-c54490b81e18
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Study 2] 077550453c71bf3e748d_2 f0148a44-4112-4e6d-b233-461657778d2d
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Study 3] 077550453c71bf3e748d_3 9bdafa26-a09f-4152-96bc-601dc04ada06
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Study 4] 077550453c71bf3e748d_4 7d212f52-4952-4368-8fae-2d032535ada5
[Homebridge] [1/3/2020, 10:01:35 AM] [TuyaWebPlatform] Configuring cached accessory [Study USB] 077550453c71bf3e748d_7 0d2a7428-49e3-4207-b19d-f2a1a789a796
[Homebridge] [1/3/2020, 10:01:36 AM] [TuyaWebPlatform] Initializing TuyaWebPlatform...
[Homebridge] [1/3/2020, 10:01:36 AM] API launched
[Homebridge] [1/3/2020, 10:01:37 AM] Service is running on port 51826.
[Homebridge] [1/3/2020, 10:01:40 AM] [MyQ2] Initializing platform accessory 'Mom’s Garage Door (ID: 7672531)'...
[Homebridge] [1/3/2020, 10:01:40 AM] [MyQ2] Initializing platform accessory 'Dad’s Garage Door (ID: 9627410)'...
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Family 1 (switch / 077550453c71bf3e720c_1)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Family 1] [077550453c71bf3e720c_1] [e4ce4d75-fbd4-4914-bcb0-3fb3b17cf474]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Family 2 (switch / 077550453c71bf3e720c_2)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Family 2] [077550453c71bf3e720c_2] [c1a505da-7679-4e43-aac0-5de467bc6849]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Family 3 (switch / 077550453c71bf3e720c_3)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Family 3] [077550453c71bf3e720c_3] [af6bad5e-6213-4dde-8e94-0704e7e7a132]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Family 4 (switch / 077550453c71bf3e720c_4)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Family 4] [077550453c71bf3e720c_4] [cd6a38e7-4e1a-45b1-9b1e-5645456efce2]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Family USB (switch / 077550453c71bf3e720c_7)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Family USB] [077550453c71bf3e720c_7] [8b7507af-af87-4f20-9798-64a8f0197a27]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Henry 1 (switch / 04200320dc4f222e943c_1)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Henry 1] [04200320dc4f222e943c_1] [2d0c8c38-9f5b-410a-b26f-5a14a833d445]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Henry 2 (switch / 04200320dc4f222e943c_2)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Henry 2] [04200320dc4f222e943c_2] [d7d6a5ab-86ee-40e5-a67a-19e62fd89ff0]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Henry 3 (switch / 04200320dc4f222e943c_3)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Henry 3] [04200320dc4f222e943c_3] [0e9e2b27-8d89-4c4a-bf02-2a3a56df1179]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Henry 4 (switch / 04200320dc4f222e943c_4)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Henry 4] [04200320dc4f222e943c_4] [a5edeb6b-b0e0-4923-ad20-4b66f94b84e7]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Henry USB (switch / 04200320dc4f222e943c_5)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Henry USB] [04200320dc4f222e943c_5] [05936f33-4676-4519-a53d-1d9fef5ca9dc]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Study 1 (switch / 077550453c71bf3e748d_1)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Study 1] [077550453c71bf3e748d_1] [876a3c05-666a-4103-87cb-c54490b81e18]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Study 2 (switch / 077550453c71bf3e748d_2)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Study 2] [077550453c71bf3e748d_2] [f0148a44-4112-4e6d-b233-461657778d2d]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Study 3 (switch / 077550453c71bf3e748d_3)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Study 3] [077550453c71bf3e748d_3] [9bdafa26-a09f-4152-96bc-601dc04ada06]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Study 4 (switch / 077550453c71bf3e748d_4)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Study 4] [077550453c71bf3e748d_4] [7d212f52-4952-4368-8fae-2d032535ada5]
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Adding: Study USB (switch / 077550453c71bf3e748d_7)
[Homebridge] [1/3/2020, 10:01:42 AM] [TuyaWebPlatform] Existing Accessory found [Study USB] [077550453c71bf3e748d_7] [0d2a7428-49e3-4207-b19d-f2a1a789a796]
[Homebridge] [1/3/2020, 10:01:47 AM] [TplinkSmarthome] New Device Online: [Smart Outlet] plug [80062BDC6455BAAC551F9C6E9458DE991BA17CD8] 192.168.0.79 9999
[Homebridge] [1/3/2020, 10:01:47 AM] [TplinkSmarthome] Adding: [Smart Outlet] plug [80062BDC6455BAAC551F9C6E9458DE991BA17CD8]
[Homebridge] [1/3/2020, 10:01:48 AM] [TplinkSmarthome] New Device Online: [Kitchen Under-Cabinet Lights] plug [8006F2667FD8CB480360E856E8B0C0E11B6D1481] 192.168.0.78 9999
[Homebridge] [1/3/2020, 10:01:48 AM] [TplinkSmarthome] Adding: [Kitchen Under-Cabinet Lights] plug [8006F2667FD8CB480360E856E8B0C0E11B6D1481]
[Homebridge] [1/3/2020, 10:01:48 AM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 10:01:49 AM] [MyQ2] Mom’s Garage Door is closed.
[Homebridge] [1/3/2020, 10:01:59 AM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 10:02:00 AM] [MyQ2] Mom’s Garage Door is closed.
[Homebridge] [1/3/2020, 10:02:14 AM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 10:02:14 AM] [MyQ2] Mom’s Garage Door is closed.
[Homebridge] [1/3/2020, 10:02:55 AM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 10:02:56 AM] [MyQ2] Mom’s Garage Door is closed.
[Homebridge] [1/3/2020, 11:27:55 AM] [MyQ2] Dad’s Garage Door is open.
[Homebridge] [1/3/2020, 11:28:56 AM] [MyQ2] Dad’s Garage Door is closing.
[Homebridge] [1/3/2020, 11:29:07 AM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 11:43:01 AM] [TuyaWebPlatform] Error retrieving devices states { Error: connect ECONNREFUSED 192.168.0.64:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1106:14)
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '192.168.0.64',
  port: 443 }
[Homebridge] [1/3/2020, 11:43:11 AM] [MyQ2] Error polling MyQ servers: FetchError: request to https://myqexternal.myqdevice.com/api/v4/UserDeviceDetails/Get failed, reason: connect ECONNREFUSED 192.168.0.64:443
[Homebridge] [1/3/2020, 11:43:12 AM] [TuyaWebPlatform] Error retrieving devices states { Error: connect ECONNREFUSED 192.168.0.64:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1106:14)
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '192.168.0.64',
  port: 443 }
[Homebridge] [1/3/2020, 11:43:31 AM] [TuyaWebPlatform] Error retrieving devices states { Error: connect ECONNREFUSED 192.168.0.64:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1106:14)
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '192.168.0.64',
  port: 443 }
[Homebridge] [1/3/2020, 3:32:21 PM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 3:32:21 PM] [MyQ2] Mom’s Garage Door is closed.
[Homebridge] [1/3/2020, 3:53:59 PM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 3:54:00 PM] [MyQ2] Mom’s Garage Door is closed.
[Homebridge] [1/3/2020, 3:56:46 PM] [MyQ2] Mom’s Garage Door is closed.
[Homebridge] [1/3/2020, 3:56:47 PM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 3:59:46 PM] [MyQ2] Dad’s Garage Door is closed.
[Homebridge] [1/3/2020, 3:59:46 PM] [MyQ2] Mom’s Garage Door is closed.
```

**Additional context**

**Comment by mkellsy** [comment]
npm install is cpu intensive, you may get spikes when upgrading and cpu's don't cool right away, it takes time. 100 - 120 degrees F is normal operating temperature depending on the temperature in the room.

You need to start worrying around 176 degrees F (80c). Remember room temperature is around 72 F.

**Comment by gleeglee217** [comment]
> npm install is cpu intensive, you may get spikes when upgrading and cpu's don't cool right away, it takes time. 100 - 120 degrees F is normal operating temperature depending on the temperature in the room.
> 
> You need to start worrying around 176 degrees F (80c). Remember room temperature is around 72 F.

Hi I would like to chime in on high temps and read that the Pi 4 had an issue with high temperatures as indicated on: 
https://www.raspberrypi.org/blog/thermal-testing-raspberry-pi-4/ 

So this makes me wonder if I should use a microSD card loaded with NOOBS to update the Pi firmware BEFORE I install HOOBS? I searched and can not find an answer. Please advise thanks.

**Comment by mkellsy** [comment]
To upgrade the firmware (same as updating Debian/Ubuntu to the latest version)
```
sudo apt update
sudo apt full-upgrade
```

When I build the HOOBS SD cards, each version pulls from the Raspbian sources. So it is pulling the latest version of the software.

I will check this further, but I suspect a `sudo apt full-upgrade` won't actually do anything with the HOOBS image.

I will test this further.

**Comment by mkellsy** [comment]
To further this issue. I found something in the code. The terminal was being started when HOOBS starts. This should only start then you go to the Terminal screen. And then close when you leave the screen.

I fixed this on 3.1.19

**Comment by gleeglee217** [comment]
> To upgrade the firmware (same as updating Debian/Ubuntu to the latest version)
> 
> ```
> sudo apt update
> sudo apt full-upgrade
> ```
> 
> When I build the HOOBS SD cards, each version pulls from the Raspbian sources. So it is pulling the latest version of the software.
> 
> I will check this further, but I suspect a `sudo apt full-upgrade` won't actually do anything with the HOOBS image.
> 
> I will test this further.

So I will be receiving a Pi 4 kit from Vilros soon and it comes with NOOBS micrSd card so should I use that to update Pi firmware FIRST and then replace it with a HOOBS microSd ?

**Comment by mkellsy** [comment]
The Raspberry Pi's firmware is NOOBS or Raspbian. HOOBS is built on Raspbian. We use the exact same software to build HOOBS the Raspberry Pi foundation uses to build NOOBS or Raspbian. All 3 can run these commands.
```
sudo apt update
sudo apt full-upgrade
```
In fact with HOOBS every version we release uses the latest sources, so running a full-upgrade is redundant.

**Comment by gleeglee217** [comment]
> The Raspberry Pi's firmware is NOOBS or Raspbian. HOOBS is built on Raspbian. We use the exact same software to build HOOBS that the Raspberry Pi foundation uses to build NOOBS or Raspbian. All 3 can run these commands.
> 
> ```
> sudo apt update
> sudo apt full-upgrade
> ```
> 
> In fact with HOOBS every version we release uses the latest sources, so running a full-upgrade is redundant.
> 
> All we do is this.
> 
> 1. Install Node
> 2. Install HOOBS
> 3. Add WiFi connect
> 
> The pi 4 is as simple as this
> https://github.com/hoobs-org/HOOBS/wiki/2.2-HOOBS-SD-Card
> 
> PS. I use a Raspberry Pi 4 for HOOBS. And I test new releases on the 4B before testing on the 3B.

Oh I see. Well I am totally new to this but I am planning to use Wyze cam v2 with Hoobs/Homebridge but I am clueless on how/where to start. There is an article: 
https://medium.com/dirigible/wyze-cam-homekit-58c4878c4124

But it seems daunting for a beginner like me so is there an easier way to set up Wyze with Hoobs? Thanks again 😊

**Comment by mkellsy** [comment]
We are working on compiling ffmpeg directly into the SD card images for raspberry pi 3B, 3B+ and 4B. You will need to re-flash if you don't want to run the commands.

**Comment by mkellsy** [comment]
There are better "Homebridge" specific instructions here
https://github.com/dgreif/ring/wiki/FFmpeg#homebridge-audio-support

**Comment by mkellsy** [comment]
We now have ffmpeg built directly into the 3.1.20 ARM 7 image.

If you don't want to reflash you can install the pre-compiled ffmpeg with this command.
```
wget -q -O - http://bit.ly/get-hoobs | sudo bash -
```

Note. This will only work on Raspbian on a newer pi with the armv7l architecture.

**Comment by gleeglee217** [comment]
Thanks for the quick reply, 

1) so every Hoobs release already includes the latest Pi firmware correct?

2) 3.1.20 includes Pi firmware 137ab correct? 

3) Btw how can I check on current firmware on Hoobs?
sudo apt -y install rpi-eeprom;
then
sudo rpi-eeprom-update;
Do I even need eeprom for Hoobs?

4) so 3.1.20 includes ffmpeg but I am still clueless on how to use Wyze cam with Hoobs 😊

**Comment by rvs007** [comment]
> We now have ffmpeg built directly into the 3.1.20 ARM 7 image.
> 
> If you don't want to reflash you can install the pre-compiled ffmpeg with this command.
> 
> ```
> wget -q -O - http://bit.ly/get-hoobs | sudo bash -
> ```
> 
> Note. This will only work on Raspbian on a newer pi with the armv7l architecture.

I just got myself a Raspbi4b and loaded HOOBS on it and started installing plugins.  HOOBS updated the version to 3.1.20 after it booted up on the Raspi4.  I came across this post and wanted to add my Wyze cams to homekit so I executed the wget command in the terminal.  Is there anything else I need to install?  I followed [this link](https://medium.com/dirigible/wyze-cam-homekit-58c4878c4124) and added one of my wyze cams to homebridge and added it in the home.app.  But the camera feed doesn't work.  I made sure the rtsp line is correct with the correct login & password and pointed to the right ip address.  Is there anything else I need to install in Raspian?
